sdk/textanalytics/Azure.AI.TextAnalytics/src/autorest.md

```diff
@@ -7,7 +7,7 @@ Run `dotnet build /t:GenerateCode` to generate code.
 
 ``` yaml
 input-file:
-    - https://raw.githubusercontent.com/Azure/azure-rest-api-specs/d89b288853959fde45622730995d5c403f332164/specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.2-preview.2/TextAnalytics.json
+    - https://raw.githubusercontent.com/Azure/azure-rest-api-specs/5efb2eca2fc3d94f27015f5d3176786c7497f946/specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.2-preview.2/TextAnalytics.json
```

Comment:
seems this is not the latest swagger hash,
https://raw.githubusercontent.com/Azure/azure-rest-api-specs/71c9d235dab9206194691d083f0248c8613e2e17/specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.2-preview.2/TextAnalytics.json